lib/spack/spack/fetch_strategy.py

```diff
@@ -152,7 +152,7 @@ def archive(self, destination):
         """
 
     @property
-    def cachable(self):
+    def cacheable(self):
```

Comment:
Let's just do comments and docstrings in this PR, and avoid functions/methods/classes